stacker/actions/build.py

```diff
@@ -274,18 +275,32 @@ def _launch_stack(self, stack, **kwargs):
             return SubmittedStatus("re-creating stack")
         elif not provider_stack:
             logger.debug("Creating new stack: %s", stack.fqn)
-            self.provider.create_stack(stack.fqn, template, parameters,
-                                       tags)
+            if force_change_set:
```

Comment:
This is a little confusing - I think it likely makes more sense to have this happen inside `create_stack` - maybe provide a flag there that will allow `create_stack` to use changesets with the `CREATE` `ChangeSetType`?